### ISSUE TITLE
Author list: accented letters are not grouped with non-accented

### ISSUE THREAD

**Issue by gczobel** [seblucas/cops#298]
Hi,
Some authors with an spanish last name have accented characters in the first letter. Example: "Úbeda, Francisco López de"
Not sure about other languages but in Spanish accented letters must be grouped with non accented ones.
So... á = a, é = e, í = i, ó = o and ú = u

In the authors list, I see different entries for authors with accented and non-accented first letter.

**Comment by seblucas** [comment]
It's a bug indeed but not one that can be easily fixed. 

You can check https://github.com/seblucas/cops/issues/262#issuecomment-211400043 to see it's a long time problem. It would need specific extension to sqlite which are hard to handle with PHP.

In France, we also have diacritics (accentuated characters and some harder cases) and I have the same sorting problem. The only solution I have for now is to use https://github.com/seblucas/cops/blob/master/config_default.php#L291-L297. You won't see any accentuated anymore in your list but the list will be correctly sorted. But this parameter is really slow.

**Comment by Ecksters** [comment]
Perhaps [Collator('bg_BG')](http://stackoverflow.com/questions/120334/how-to-sort-an-array-of-utf-8-strings)  might be a potential general solution, still not perfect though.

**Comment by seblucas** [comment]
That's true that I could sort the array in PHP (for now every sort was done by the database)

**Comment by Ecksters** [comment]
SQLite3 is supposed to support UTF8 encoding, and supposedly should sort it correctly, are we using that?

**Comment by Rurik19** [comment]
I have suggestion - calibre has "sort_name" in the authors table. May be we just can store non-accented letters in that field an use it to the sorting?

**Comment by seblucas** [comment]
@Ecksters 

Thanks for the idea with Collator, it seems to work with French, I'll check the performance hit and will fix it soon

**Comment by gczobel** [comment]
Something new about this?
